recipe/meta.yaml

```diff
@@ -12,17 +12,17 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1000
```

Comment:
sorry this should be 0 (I didn't realize that you haven't been migrated yet, we haven't rebuilt `ffmpeg` yet)

Reply:
Oh, no worries! Transitive dependencies are the bane of my Gentoo existence. Let's revert that.